book/tutorials/database/1_getting_started_example.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to the SnowEx Database "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "## Why a database?\n",
+    "> *<span style=\"font-size:12pt;\">\"Dude, I am into pits not bits. What gives?!\"</span>*\n",
+    "\n",
+    "- Standardizing diverse datasets\n",
+    "- Cross referencing data\n",
+    "- Enables GIS functionality\n",
+    "- Ready for use in your code\n",
+    "- Provenance!\n",
+    "- Ready for use in a GIS software like ArcGIS or QGIS!\n",
+    "\n",
+    "### TL;DR Do less wrangling, do more crunching. \n",
+    "\n",
+    "\n",
+    "# What is it exactly?\n",
```

Comment:
In order for the sidebar navigation to work as expected you want only one header per file and the rest subheaders:
```
```
# Introduction to the SnowEx Database
## Why a database?
### TL;DR Do less wrangling, do more crunching.
## What is it exactly?
```
```